backend/ee/danswer/external_permissions/slack/group_sync.py

```diff
@@ -0,0 +1,89 @@
+"""
```

Comment:
this comment is correct. Just keeping this here in casse we add support for group based stuff in the future. It's tested to work